docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -614,33 +614,64 @@ with the group quarters person as the reference person, and that GQ "household"
-the (weighted) proportion of ACS PUMS persons that are recent immigrants consistent with that move type.
+the (weighted) proportion of ACS PUMS persons in the simulation catchment area that are recent immigrants consistent with that move type.
+Since immigration is likely unaffected by US population change over time, the number of immigrants for a move type
+is the rate multiplied by the simulation's **initial/configured** population size, not current population size.
 At each time step:
 
-#. ACS PUMS households with reference people who are recent immigrants, after removing household members who are not recent immigrants, (as well as GQ "households" of recent immigrants)
-   are sampled using household weights (equal to person weights in the case of GQ) with replacement until the desired number of simulants added in household moves is reached.
-#. ACS PUMS recent immigrants living in residential households where the reference person is not a recent immigrant are sampled using person weights with replacement until the desired
-   number of simulants added in non-reference-person moves is reached.
+#. ACS PUMS households with reference people who are recent immigrants (as well as GQ "households" where the GQ person is a recent immigrant),
+   after removing any household members who are not recent immigrants,
+   are sampled using household weights (equal to person weights in the case of GQ).
+   This sampling continues with replacement until the desired number of simulants added in household moves is reached.
+#. ACS PUMS recent immigrants living in residential households where the reference person is not a recent immigrant are sampled using person weights.
+   This sampling continues with replacement until the desired number of simulants added in non-reference-person moves is reached.
 
-Added residential households are assigned a new household ID, as is done at population initialization.
+Added residential households are assigned a new household ID and a new address, as is done at population initialization.
 Added GQ people (who all enter in "household" moves) should be assigned a household ID for a randomly-selected GQ type matching
-their institutional/non-institutional status, as is done at population initialization.
+their institutional/non-institutional status, as well as the corresponding shared address, as is done at population initialization.
+
+Additionally, we perturb the PUMA attribute of the sampled person (in the case of a non-reference-person move) or household (in the case of a household move).
+This is intended to ensure a non-zero probability of immigration into all PUMAs,
+and to lower the chances of very similar simulants being sampled from the same ACS person/household.
+The perturbation process is as follows:
+**10%** of the time, replace the sampled person/household's PUMA with a PUMA selected at random according to the
+probabilities in the appropriate file below.
```

Comment:
following these probabilities, do we expect there is any chance we choose a PUMA with no other households? In the case that we do that for a non-household head move, is that okay, and we make them a household head? or that shouldn't have been possible?